app.js

```diff
@@ -0,0 +1,86 @@
+const School = require('./School.js');
+const Student = require('./Student.js');
+const express = require('express');
+const cors = require('cors');
+const bodyParser = require('body-parser');
+
+const port = 4000;
+const app = express();
+app.use(cors());
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({extended: false}));
+
+
+// const classRouter = require("./Class.js");
+// const schoolRouter = require("./School.js");
+// const studentRouter = require("./Student.js");
+
+// app.use("/class", classRouter);
+// app.use("/School", schoolRouter);
+// app.use("/Student", studentRouter);
+
+let mySchool = new School();
+
+app.get("/", (req, res) => {
+    try{
+        res.json(mySchool);
+    } catch(error) {
+        console.log(error);
+    }
+});
+
+app.post("/class", (req, res) => {
```

Comment:
You are not accounting for the existing class here. You should use a middleware function (ex. checkIfClassExists) then would send a response message if the class does already exist, or move on to the function of actually adding the class using `next()` .